docs/source/reference/command_line.rst

```diff
@@ -313,6 +313,43 @@ Below is a list with all available subcommands.
       list  Display a list of all available plugins.
 
 
+.. _reference:command-line:verdi-presto:
+
+``verdi presto``
+----------------
+
+.. code:: console
+
+    Usage:  [OPTIONS]
+
+      Set up a new profile as automatically as possible.
+
+      This command aims making setting up a new profile as easy as possible. It intentionally
+      provides a limited amount of options to customize the profile and by default does not
+      require any options to be specified at all. For full control, please use the command
+      `verdi profile setup` instead.
+
+      The main goal for this command is to setup a profile with minimal requirements to make
+      it easy to install AiiDA and get started as quickly as possible. Therefore, by default,
+      the created profile uses the `core.sqlite_dos` storage plugin which does not require any
+      services, such as PostgreSQL and RabbitMQ are not required. This _does_ mean, however,
+      that not all functionality of AiiDA is available, most notably running the daemon and
+      submitting processes to said daemon.
```

Comment:
There is some repetition with respect with the previous paragraph, suggested rewording.

```suggestion
      After running `verdi presto` you can immediately start using AiiDA without additional setup. 
      The created profile uses the `core.sqlite_dos` storage plugin which does not require any
      services, such as PostgreSQL and RabbitMQ. This _does_ mean, however,
      that not all functionality of AiiDA is available, most notably running the daemon and
      submitting long-running processes to said daemon.
```

Reply:
I removed "long-running" processes from your suggestion. I get the point, but it suggests that some other processes might be submitted, but there is no submission whatsoever.

Reply:
Makes sense. Although I just realized that the given text is still not quite accurate, since we're mixing the DB plugin (which is just a question of performance) with the presence of rabbitmq, which is what actually determines whether we can have a darmon or not! And now you're even detecting whether rabbitmq is present, and if it is, all is actually fine. So I think a bit more nuance here is needed.

Reply:
The plan is to extend this command with the functionality of `verdi quicksetup` to automatically create resources needed for the `core.psql_dos`. So the comment will become more incorrect. But I will already update it now for the fact that it tries to configure RabbitMQ if it can connect to it. BEcause that is already in